opencti-platform/opencti-graphql/src/modules/case/case-task/case-task.ts

```diff
@@ -0,0 +1,48 @@
+import { v4 as uuid } from 'uuid';
+import { ENTITY_TYPE_CONTAINER } from '../../../schema/general';
+import { normalizeName } from '../../../schema/identifier';
+import { ModuleDefinition, registerDefinition } from '../../../schema/module';
+import { objectAssignee } from '../../../schema/stixRefRelationship';
+import convertCaseTaskToStix from './case-task-converter';
+import caseTaskResolvers from './case-task-resolvers';
+import type { StixCaseTask, StoreEntityCaseTask } from './case-task-types';
+import { ENTITY_TYPE_CONTAINER_CASE_TASK } from './case-task-types';
+import caseTaskTypeDefs from './case-task.graphql';
+
+const CASE_TASK_DEFINITION: ModuleDefinition<StoreEntityCaseTask, StixCaseTask> = {
+  type: {
+    id: 'case-task',
+    name: ENTITY_TYPE_CONTAINER_CASE_TASK,
+    category: ENTITY_TYPE_CONTAINER
```

Comment:
Does it appear in the list of entity settings?
Is this what we want? (we can customize workflow)

Reply:
Yes ? We want to customize workflow of tasks